pylearn2/termination_criteria/__init__.py

```diff
@@ -221,16 +221,16 @@ def initialize(self, model):
         if self._new_epochs:
             self._epochs_done = 0
```

Comment:
@lamblin Do you think it's correct? If they use transfer_experience=False they should not want to preserve previous number of epochs so new_epochs=False wouldn't make sense and will behave as new_epochs=True anyway.